lib/fs/copy-symlink.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const nativeFs = require('fs');
+const path = require('path');
+
+const promisify = require('es6-promisify');
```

Comment:
Why not [**pify**](https://github.com/sindresorhus/pify)? The pify has optimizations for v8 like native `promisify` from `util` – https://github.com/sindresorhus/pify/blob/master/optimization-test.js.